src/resources/views/components/sidebars/menu.blade.php

```diff
@@ -0,0 +1,40 @@
+@if(count($sidebarMenu->children) > 0)
+    <li class="nav-item menu-items {{ isSidebarMenuActive($sidebarMenu->slug)? 'active' : '' }}">
+        <a class="nav-link level-{{$i}}"
+            data-toggle="collapse" href="#ui-{{ $sidebarMenu->id }}"
+            aria-expanded="false" aria-controls="ui-{{ $sidebarMenu->id }}">
+            <span class="menu-icon {{ !empty($sidebarMenu->icon) ? '' : 'd-none' }}">
+                {!! $sidebarMenu->icon !!}
+            </span>
+            <span class="menu-title">{{ $sidebarMenu->name }}</span>
+            <i class="menu-arrow"></i>
+        </a>
+        <div class="collapse" id="ui-{{ $sidebarMenu->id }}">
+            <ul class="nav flex-column sub-menu">
+                @foreach($sidebarMenu->children as $children)
+                    @if(!count($children->children) > 0)
```

Comment:
có cần check chỗ này không ?

Reply:
@toixtran Đoạn này cần check bởi vì nó liên quan đến phần `if-else` to ở bên trên.